package.json

```diff
@@ -6,16 +6,19 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon server",
-    "build": "webpack -d -w"
+    "build": "webpack -d -w",
+    "seed": "cd db/ && npx sequelize db:drop && npx sequelize db:create && npx sequelize db:migrate && npx sequelize db:seed:all || npm run fallback",
+    "fallback": "cd db/ && npx sequelize db:create && npx sequelize db:migrate && npx sequelize db:seed:all"
```

Comment:
Is `fallback` handling the case where there isn't currently a table to drop in the database?

Reply:
Sorry I didn't see this before I merged the request. Yes with the "||" in the seed script, if the first script fails (most likely due to the database not existing) then the "fallback" script will run. Since seeding through sequelize migrations, it was easier than hard coding in the documents.